server/src/test/java/org/solarbank/server/unit/service/CalculateServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package org.solarbank.server.unit.service;
+
+import javax.money.CurrencyUnit;
+import javax.money.Monetary;
+import javax.money.MonetaryAmount;
+import org.javamoney.moneta.Money;
+import org.junit.jupiter.api.Test;
+import org.solarbank.server.dto.CalculateRequest;
+import org.solarbank.server.dto.CalculateResult;
+import org.solarbank.server.dto.CalculateResult.SavingsPerYear;
+import org.solarbank.server.service.CalculateService;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Map;
+
+public class CalculateServiceTest {
```

Comment:
unit tests added here. Not started the unit tests for the exceptions as they are still work in progress